_includes/tutorials/kafka-producer-application/kafka/markup/dev/make-dev-file.adoc

```diff
@@ -0,0 +1,21 @@
+Then create a development file at `configuration/dev.properties`:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-producer-application/kafka/code/configuration/dev.properties %}</code></pre>
++++++
+
+Let's do a quick walkthrough of some of the properties.
+
+https://kafka.apache.org/documentation/#key.serializer[key.serializer] - The serializer the `KafkaProducer` will use to serialize the key.
+
+https://kafka.apache.org/documentation/#value.serializer[value.serializer] - The serializer the `KafkaProducer` will use to serialize the value.
+
+https://kafka.apache.org/documentation/#acks[acks] - The `KafkaProducer` uses the `acks` configuration to tell the lead broker how many acknowledements to wait for to consider a produce request complete. Acceptable settings are `0`, `1`, `-1`, or all`.  Setting `acks` to `-1` is the same as setting it to `all`.  The default setting is `1`.
+
+
+- Setting `acks` to `0` could be considered "fire and forget" as once the producer sends the record batch it is considered succesful even though the records may not reach the broker at all. So record loss is possible.  The offsets for records are always set to `-1`.
```

Comment:
Overall the description of the various `acks` settings seems overly verbose and potentially too much to throw at a new user.  FYC make a pruned down version and link to the docs for more information.  A pruned down version could look something like this (just as an example):

```
- `acks=0`: "fire and forget", once the producer sends the record batch it is considered successful
- `acks=1`: leader broker added the records to its local log but didn't wait for any acknowledgement from the followers
- `acks=all`: highest data durability guarantee, the leader broker persisted the record to its log and received acknowledgement of replication from all in-sync replicas
```

Reply:
ack